test/propagated_params_test.go

```diff
@@ -44,7 +44,8 @@ var (
 	ignoreConditions        = cmpopts.IgnoreFields(duckv1.Status{}, "Conditions")
 	ignoreContainerStates   = cmpopts.IgnoreFields(corev1.ContainerState{}, "Terminated")
 	ignoreStepState         = cmpopts.IgnoreFields(v1beta1.StepState{}, "ImageID")
-	ignoreTaskRunSpec       = cmpopts.IgnoreFields(v1beta1.TaskRunSpec{}, "Resources")
+	ignoreTaskRunSpec       = cmpopts.IgnoreFields(v1beta1.TaskRunSpec{}, "Resources", "ServiceAccountName")
```

Comment:
Is this variable used?

Reply:
do you mean `ignoreTaskRunSpec`? it's used here - https://github.com/tektoncd/pipeline/blob/f0b8c16bbf913da1e86044109a72693c885d5c1c/test/propagated_params_test.go#L126